src/ui/mod.rs

```diff
@@ -1,19 +1,23 @@
-use app::{App, AppResult};
-use event::{Event, EventHandler};
-use handler::{handle_key_events, handle_mouse_events};
 use ratatui::{backend::CrosstermBackend, Terminal};
 use std::io;
-use tui::Tui;
 
-use crate::args::Args;
+use crate::args::*;
 
 mod app;
+use app::{App, AppResult};
```

Comment:
why do we mix `mod`'s and `use`'s.

Reply:
Because it makes more sense to group these statements like this (at least according to me), so that you can see all the reexports at the first glance. Example from another project: [ureq](https://github.com/algesten/ureq/blob/e4e7e4e51209d8a14c741ad46991cd83e1a18379/src/lib.rs#L355)